pasteit/public/javascripts/globals.js

```diff
@@ -0,0 +1,49 @@
+var languages = { 
+    "C++"             : ["clike"],
+    "C"               : ["clike"],
+    "Java"            : ["clike"],
+    "Objective-C"     : ["clike"],
+    "C#"              : ["clike"],
+    "Clojure"         : ["clojure"],
+    "Apricot"         : ["clojure"],
+    "CoffeeScript"    : ["coffeescript"],
+    "CSS"             : ["css"],
+    "Diff"            : ["diff"],
+    "Go"              : ["go"],
+    "Groovy"          : ["groovy"],
+    "Haskell"         : ["haskell"],
+    "Javascript"      : ["javascript"], 
+    "Lua"             : ["lua"],
+    "Delphi"          : ["pascal"],
+    "Perl"            : ["perl"],
+    "PHP"             : ["php"],
+    "Java Properties" : ["properties"],
+    "Python"          : ["python"],
+    "R"               : ["r"],
+    "Ruby"            : ["ruby"],
+    "Rust"            : ["rust"],
+    "Scheme"          : ["scheme"],
+    "Emacs Lisp"      : ["scheme"],
+    "Smalltalk"       : ["smalltalk"],
+    "Verilog"         : ["verilog"],
+    "XML"             : ["xml"],
+    "YAML"            : ["yaml"],
+    "HTML"            : ["htmlmixed"],
+    "MySQL"           : ["mysql"],
+    "OCaml"           : ["ocaml"],
+};
+
+var mimes = {
+    "C++"  : "text/x-c++src",
+    "C"    : "text/x-csrc",
+    "C#"   : "text/x-csharp",
+    "Objective-C" : "text/x-objectivec",
+    "Java" : "text/x-java",
+};
+
+var getLangKey = function(value) {
```

Comment:
Change the value param to something more specific. Maybe languageName?

Reply:
done in 25e11d9